### ISSUE TITLE
NullPointerException - PushPayload.java:16 - Android 4.3, API 18

### ISSUE THREAD

**Issue by seanadkinson** [intercom/intercom-android#442]
Seeing this issue in our Bugsnag:

## Version info
  - Intercom for Android version: 4.1.0
  - Android OS version: 4.3 (API 18)
  - Android Device: Nexus 5

## Stacktrace
```
java.lang.NullPointerException
    PushPayload.java:16 io.intercom.android.sdk.push.PushPayload.create
    PushHandler.java:42 io.intercom.android.sdk.push.PushHandler.isIntercomPush
    IntercomPushClient.java:169 io.intercom.android.sdk.push.IntercomPushClient.isIntercomPush
    IntercomIntentService.java:24 com.robinpowered.react.Intercom.IntercomIntentService.onHandleIntent
    IntentService.java:65 android.app.IntentService$ServiceHandler.handleMessage
    Handler.java:99 android.os.Handler.dispatchMessage
    Looper.java:137 android.os.Looper.loop
    HandlerThread.java:61 android.os.HandlerThread.run
```

Don't have much else to go on, since I can't see the source code.  Can you guys check `PushPayload.java` line 16, and let me know what could be `null`?

We are using android support libraries version 26.1.0.  Let me know if I can provide more info, thanks.

**Comment by ronocod** [comment]
Thanks, we're working on a fix for this now.

**Comment by ronocod** [comment]
This is fixed in 4.1.3.

**Comment by seanadkinson** [comment]
Thank you
